content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -103,6 +103,17 @@ The following methods exist for installing kubectl on macOS:
    sudo chown root: /usr/local/bin/kubectl
    ```
 
+   {{< note >}}
+   For more information about what PATH is, refer to [Set or modify a path in Unix](https://kb.iu.edu/d/acar).
+   
+   On macOS,  `/usr/local/bin/` is usually included in your `$PATH`.
+
+   You can run the following command to detect:
+   ```bash
+   es=$(echo $PATH | grep /usr/local/bin);if [[ -z "$es" ]]; then;echo "Detected that PATH does not contain /usr/local/bin";echo "Please refer to https://kb.iu.edu/d/acar, add /usr/local/bin to the PATH";fi; echo "You can proceed to the next step";
+   ```
```

Comment:
This is an overkill. 
Just mention that users must have `/usr/local/bin` must be in their `$PATH` would be okay.
Adding scripts may introduce unnecessary problems, considering that not all users use Bash.
All new contents, once added, are to be maintained. Please bear that in mind when proposing fixes.

> For more information about what PATH is, refer to [Set or modify a path in Unix](https://kb.iu.edu/d/acar).

This is beyond what kubernetes website is about.

Reply:
TIL.